server/templates/static/about.html

```diff
@@ -103,6 +104,23 @@ <h2>Advisory Board</h2>
           Varian</a>, Chief Economist, Google. </li>
     </ul>
   </section>
+
+  <section>
+    <h2>Origins</h2>
+    <p>
+      Data Commons arose from a set of <a
+        href="https://www.nitrd.gov/nitrdgroups/index.php?title=Open_Knowledge_Network">meetings</a>
+      around this concept, organized by Chaitan Baru of NSF, Andrew Moore
+      (then of CMU) and R.V.Guha, in 2016 and 2017. That eventually lead to
```

Comment:
Space before Guha?

Reply:
that seems to be how he writes his name, so i didn't want to update it. see guha.com